packages/editor/src/components/block-mover/index.js

```diff
@@ -66,6 +66,30 @@ export class BlockMover extends Component {
 					onFocus={ this.onFocus }
 					onBlur={ this.onBlur }
 				/>
+				<Draggable
```

Comment:
Thinking it might be worth setting this as an optional prop (default `draggable`  to true) on the block mover.

Reply:
Yes, I was thinking something that would let you use the mover without drag handles for a certain UI if needed.